doc/upgrade-cluster.md

```diff
@@ -16,25 +16,12 @@ Your Bookkeeper cluster should be in a healthy state. You can check your cluster
 ```
 $ kubectl get bk
 NAME        VERSION   DESIRED MEMBERS   READY MEMBERS   AGE
-bookkeeper  0.6.1        7                 7            11m
+bookkeeper  0.4.0        7                 7            11m
```

Comment:
why is the version changed to `0.4.0`

Reply:
this is to make it consistent with the values in this dosument